index.markdown

```diff
@@ -292,23 +292,23 @@ In next sections, we will see stack opcodes that receive parameters from stack.
 
 ### `pick` (opcode `0x79`)
 
-`pick` reads `n` and copies `n`-th element to top stack. Example:
+`pick` reads and drops the number at the top of the stack, which we will call `n`, and copies the `n`-th element to the top of the stack. Example:
 
-    push1 push2 push3 push4 push2 pick
+    push1 push2 push3 push4 push1 pick
```

Comment:
Why change the example?

Reply:
With the previous example the stack just remains the same after applying pick, this behaviour confused me a bit when following the tutorial so I decided to change it to an example that modified the stack so other people could appreciate the behaviour of pick better.

Reply:
Still, it may be easier for some people to follow the previous example so if this change is controversial i can revert it.

Reply:
> With the previous example the stack just remains the same after applying pick

Really? :thinking: I have to check...

Reply:
![Screenshot](https://user-images.githubusercontent.com/32309574/60634868-1abb7100-9e43-11e9-8cc4-faa4c03def65.png)
After applying pick:
![Screenshot](https://user-images.githubusercontent.com/32309574/60634867-1a22da80-9e43-11e9-8dc9-ff5e874605b4.png)